src/Entity/SocialAuth.php

```diff
@@ -30,6 +31,40 @@ class SocialAuth extends ContentEntityBase implements ContentEntityInterface {
   /**
    * Creating fields.
    */
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getToken() {
+    return $this->get('token')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getData() {
+    return $this->get('additional_data')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setToken($token) {
+    $this->set('token', $token);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
```

Comment:
This is not inherited, so inheritdoc does not apply.

Change the method to setAdditionalData